policy-service/src/main/resources/application.yaml

```diff
@@ -12,27 +12,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+spring:
+  profiles:
+    include:
+    - akka
+    - default-config
+
+---
 
 spring:
+  profiles: default-config
   application:
     name: policy-service
   output:
     ansi:
       enabled: always
   main:
     allow-bean-definition-overriding: true
-  cloud:
-    loadbalancer:
-      ribbon:
-        enabled: false
+  autoconfigure:
+    exclude:
+    - org.springframework.boot.autoconfigure.data.redis.RedisAutoConfiguration
```

Comment:
This should preferably be in the caffeine config

Reply:
updated